prod/prod_test.go

```diff
@@ -31,11 +32,13 @@ import (
 	psstest "github.com/ethersphere/swarm/pss/testing"
 	"github.com/ethersphere/swarm/pss/trojan"
 	"github.com/ethersphere/swarm/storage"
+	"github.com/ethersphere/swarm/storage/feed"
 )
 
 // TestRecoveryHook tests that NewRecoveryHook has been properly invoked
 func TestRecoveryHook(t *testing.T) {
 	ctx := context.TODO()
+	ctx = context.WithValue(ctx, "publisher", "0226f213613e843a413ad35b40f193910d26eb35f00154afcde9ded57479a6224a")
```

Comment:
not sure we need `TODO()` here, could we use `Background().WithValue` maybe?

anyway this can be condensed into 1 line rather than 2 assignments

Reply:
`context.Background().WithValue(ctx, "publisher", "0226f213613e843a413ad35b40f193910d26eb35f00154afcde9ded57479a6224a")`

This does not compile `context.Background().WithValue undefined (type context.Context has no field or method WithValue)`

Maybe I misunderstood, but It does not let me merge it into one-liner directly

Reply:
nevermind, let's just call `.WithValue` directly